### ISSUE TITLE
Placeholder text changes to first selected option

### ISSUE THREAD

**Issue by floorish** [monterail/vue-multiselect#208]
If I use the props `:searchable="false"` and `:multiple="true"` the placeholder text is replaced by the label of the first chosen option.

### Steps to reproduce:
1. Define a multiselect component
````
<multiselect v-model="myModel" :options="myOptions" track-by="id" label="content" :searchable="false" :multiple="true" placeholder="myPlaceholder"></multiselect> 
````

2. Select the first option.

### Expected
placeholder stays "myPlaceholder".

### Actual
placeholder changes to whatever option you have chosen first

Introduced in PR: https://github.com/monterail/vue-multiselect/pull/164
(Before that it worked fine)

**Comment by shentao** [comment]
Thanks! I hope to fix this in the next 2 days!

**Comment by shentao** [comment]
@floorish beta.14 released and should fix this problem. Sorry you had to wait. Please read: https://github.com/monterail/vue-multiselect/blob/2.0/README.md
Breaking change regarding the styles.

**Comment by floorish** [comment]
Thanks, you're awesome!
It looks like c3fd85744c62a421bbf51b807eca1f21b9810cfd also fixes #207

Will try it soon.

**Comment by shentao** [comment]
Might be. I just went on a issue-closing spree. The release was quite large to be honest. Also planning on changing the release flow to something like stable release + nightly/canary builds that are autoreleased. That should improve the time-to-release.
